packages/jss-plugin-nested/src/index.js

```diff
@@ -98,13 +98,16 @@ export default function jssNested(): Plugin {
         container.addRule(selector, style[prop], {...options, selector})
       } else if (isNestedConditional) {
         // Place conditional right after the parent rule to ensure right ordering.
-        container
-          .addRule(prop, {}, options)
-          // Flow expects more options but they aren't required
-          // And flow doesn't know this will always be a StyleRule which has the addRule method
-          // $FlowFixMe[incompatible-use]
-          // $FlowFixMe[prop-missing]
-          .addRule(styleRule.key, style[prop], {selector: styleRule.selector})
+        const queue = container.prepareQueue()
```

Comment:
I think that this code exposes too much internal API, it probably needs to be changed.

Reply:
yeah, definitely, need to find a cleaner way

Reply:
The logic with the queue is a mess :)

Reply:
It wasn't super lean before, but now with queue from the instance and from an argument in deployRule, it became even worse